src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -68,12 +70,24 @@ public BinlogConnectorReplicator(
 			EventDeserializer.CompatibilityMode.CHAR_AND_BINARY_AS_BYTE_ARRAY);
 		this.client.setEventDeserializer(eventDeserializer);
 
-		this.binlogEventListener = new BinlogConnectorEventListener(client, queue);
+		Timer replicationQueueTimer = metrics.getRegistry().timer(metrics.metricName("replication", "queue", "time"));
+		this.binlogEventListener = new BinlogConnectorEventListener(client, queue, replicationQueueTimer);
+
 		this.client.setBlocking(!stopOnEOF);
 		this.client.registerEventListener(binlogEventListener);
 		this.client.setServerId(replicaServerID.intValue());
 
 		this.stopOnEOF = stopOnEOF;
+
+		metrics.getRegistry().register(
+				metrics.metricName("replication", "queue", "count"),
+				new Gauge<Long>() {
+					@Override
+					public Long getValue() {
+						return (long) queue.size();
```

Comment:
It doesn't tell us much about the DB, the important thing is whether maxwell is keeping up. i.e. when we see lag, is it because the DB is not sending binlog events quickly or because maxwell is too busy to read them?

I'm imagining this metric would dither between "almost 0" and "20", so maybe the actual number isn't important. It probably doesn't add anything over the wait time metric, so I'll nix it.